packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -45,7 +49,7 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.4.1
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 11.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
```

Comment:
@theforeman/builder which is packaged is 10.1.0: https://github.com/theforeman/foreman-packaging/blob/rpm/3.3/packages/foreman/nodejs-theforeman-builder/builder-10.1.0.tgz